money-core/src/main/scala/com/comcast/money/metrics/MetricRegistryFactory.scala

```diff
@@ -35,18 +35,19 @@ import scala.util.{ Try, Success, Failure }
  * Note: This trait should be kept as simple as possible so that the resulting interface can also be implemented
  * by a Java client custom factory.
  */
-object MetricRegistryFactory {
-  private val logger = LoggerFactory.getLogger("com.comcast.money.metrics.MetricRegistryFactory")
 
-  def metricRegistry(config: Config): MetricRegistry = {
+class MetricRegistryExtensionImpl(config: Config) extends Extension {
+  private val logger = LoggerFactory.getLogger("com.comcast.money.metrics.MetricRegistryExtension")
```

Comment:
should this be: getLogger("com.comcast.money.metrics.MetricRegistryExtensionImpl")